src/common/sync-box/channel/channel-listener-injection-token.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectionToken } from "@ogre-tools/injectable";
+
+export interface ChannelListener {
+  channel: any;
+  handler: (value: any) => void;
```

Comment:
I do not like that this is not type safe

Reply:
Typings for these came in latter commit. Sorry for not rebasing.